gen-compdocs/generators/doc.go

```diff
@@ -79,15 +79,18 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 	}
 
 	//testing tool-reference template (kb)
-    	if with_title {
+	if with_title {
 		if _, err := fmt.Fprintf(w, "---\ntitle: %s\ncontent_template: templates/tool-reference\nweight: 28\n---\n\n", name); err != nil {
 			return err
 		}
 
 		if _, err := fmt.Fprintf(w, "%s\n\n", "{{% capture synopsis %}}"); err != nil {
 			return err
 		}
-		
+
+		// Escape any '<', '>' characters found in the long description
+		long = strings.Replace(long, "<", "&lt;", -1)
+		long = strings.Replace(long, ">", "&gt;", -1)
```

Comment:
does `>` need an escape? i thought escaping `<` was enough.

Reply:
I think the best practice would be escaping both.

Reply:
I think there are a couple of ways to fix this, depending on what is generated in this script:
1) calling a single method on the entire string (long description) to escape ALL chars that need escaping -- is there a Go library to do this?
OR
2) leaving the string alone here and test if we can clean up the string (long description) using Hugo.

Reply:
hello @tengqm. Does this update need further review? What about adding a single call,
fmt.Println(html.EscapeString(s)), to take care of all the html chars?

Reply:
Before conversion, the input data are scattered in many different elements in the specification. This means we don't have a HTML to perform this operation. After the conversion, the whole doc is a HTML string, which is not meant to be escaped. The only thing we can do, as I see it, is somewhere during the conversion. We inject the customization logic wherever appropriate.

Did I capture what you said?

Reply:
Yes, this change seems fine, though I did not get a chance to verify. LGTM. Thanks.